src/containers/ImportResources/ImportResources.js

```diff
@@ -107,6 +110,17 @@ export class ImportResources extends Component {
     }
   };
 
+  handleInstallNamespace = ({ selectedItem }) => {
```

Comment:
how about 'importerNamespace' instead of 'installNamespace' to avoid confusion with the Dashboard install namespace?

Reply:
Sounds better